pb/summary/summary.proto

```diff
@@ -110,6 +114,8 @@ message HealthinessInfo {
   // The flakiness out of 100 (think percentage but drop the sign)
   float average_flakiness = 4;
 
+  // The average flakiness for previous intervals
+  repeated float previous_flakiness = 5;
```

Comment:
Why is this in two locations? Why does this need to be repeated? How will we decouple this value from the highly variable update cycle?

It seems like a timeseries database would be a better way to store and measure this sort of things.

Reply:
The two locations are overall vs. per-test
And this is per-interval, so it shouldn't be dependent on exactly when the cycle updates, but on past time intervals.

Are you suggesting to remove the idea of a previous flakiness number altogether, or to make it non-repeated and store this in a database for retrieval?

Reply:
(To clarify, my thought on this field is that this could have things like "current flakiness in last X days" and also "flakiness from X to 2X days ago", "flakiness from 2X to 3X days ago", etc, not necessarily things spanning longer than the history of the tab. (For longer historical data, timeseries database seems fine)